tools/BuildPackagesTask/Microsoft.Azure.Build.Tasks.Test/TestSetGeneratorTester.cs

```diff
@@ -0,0 +1,513 @@
+﻿//
+// Copyright (c) Microsoft.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+using System;
+using Xunit;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.WindowsAzure.Build.Tasks
+{
+    public class TestSetGeneratorTester
+    {
+        public const string CheckIn = "CheckIn";
+        public const string AcceptanceType = "AcceptanceType";
+        public const string mapName = "TestMappings.json";
+
+        public string GetMapFilePath()
+        {
+            var assemblyLocation = new Uri(Assembly.GetExecutingAssembly().CodeBase).LocalPath;
```

Comment:
Note that you can make this a static property initialized once rather than generating on each invoication